src/rgw/rgw_admin.cc

```diff
@@ -2021,6 +2023,7 @@ int main(int argc, char **argv)
   int sync_stats = false;
   int reset_regions = false;
   int bypass_gc = false;
+  int keep_index_consistent = false;
```

Comment:
@prallabh I'll switch the option to `--inconsistent-index`, so that by default it keeps the index consistent.